app/services/accept_auction.rb

```diff
@@ -19,7 +30,36 @@ def perform
     end
   end
 
-  private
+  def vendor_ineligible?
+    !winning_bidder_is_eligible_to_be_paid?
+  end
+
+  def winning_bidder_is_eligible_to_be_paid?
+    if auction_is_small_business?
+      reckoner = SamAccountReckoner.new(winning_bidder)
+      reckoner.set!
```

Comment:
This is basically a saving throw to see if their SAM status is good before bid right? Does the reckoner have switch from true -> false?